MetaData/data/Era2017_legacy_v1/datasets.json

```diff
@@ -3851,6 +3851,96 @@
+                "nevents": 1300
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_19.root", 
+                "nevents": 900
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_9.root", 
+                "nevents": 900
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_1.root", 
+                "nevents": 1800
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_14.root", 
+                "nevents": 18900
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_6.root", 
+                "nevents": 15300
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_7.root", 
+                "nevents": 17100
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_5.root", 
+                "nevents": 9000
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_10.root", 
+                "nevents": 25000
+            }, 
+            {
+                "name": "/store/user/alesauva/flashgg/UL2017_4/10_6_4/WminusH_HToGG_WToAll_M125_TuneCP5_13TeV-powheg-pythia8/UL2017_4-10_6_4-v0-RunIISummer19UL17MiniAOD-106X_mc2017_realistic_v6-v1/200929_090236/0000/myMicroAODOutputFile_13.root", 
+                "nevents": 20700
+            }
+        ], 
+        "parent_n_units": null
+    }, 
```

Comment:
You need to run the following command to add the status and weight for WH

`fggManageSample.py -C <Campaign> check -S <Dataset>`

Reply:
Thanks for your checks !